### DISCUSSION TITLE
click on dropdown button and open file dialog box

### DISCUSSION THREAD

**Discussion by manimovassagh** [quasarframework/quasar#14303] [General - Components / Directives / etc] [answered]
Hi everybody,
I want to make a CSV Import but on dropdown menu.
when the user click on csv open a window dialog box like import file.
something like this but on dropdown menu and not on icon
https://codepen.io/smolinari/pen/abbYdMK
i made a codepen for that and describe also inside.
Thanks for help :)
https://codepen.io/manimovassagh/pen/gOzYbjp

**Comment by pratik227** [comment, marked as answer]
https://codepen.io/Pratik__007/pen/QWrLrvy

**Reply by manimovassagh** [reply]
Thanks a lot really nice
